libs/utils/utils.js

```diff
@@ -115,6 +115,8 @@ ENVS.local = {
   name: 'local',
 };
 
+export const EVENT_MILO_DEFERRED = 'milo:deferred';
+export const EVENT_MILO_LCP_LOADED = 'milo:LCP:loaded';
```

Comment:
If this list grows with time, would it not make sense to have a single `EVENTS` object containing all events? Then you would just export one constant and you would import the same constant name wherever events are needed (instead of multiple ones if more events are required in the file).

Reply:
Good idea, will update to do that